src/services/codefixes/inferFromUsage.ts

```diff
@@ -414,27 +414,33 @@ namespace ts.codefix {
                 inferTypeFromContext(reference, checker, usageContext);
             }
             const isConstructor = declaration.kind === SyntaxKind.Constructor;

```

Comment:
This is only used in one place, you could just inline to `isConstructorDeclaration(declaration)`